lib/functions/lintly.sh

```diff
@@ -29,6 +29,12 @@ export LINTLY_SUPPORT_ARRAY                      # Workaround SC2034
 ########################## FUNCTION CALLS BELOW ################################
 ################################################################################
 ################################################################################
+#### Function AddLinterOptsForLintly ###########################################
+function AddLinterOptsForLintly() {
+  [[ ! "${LINTER_OPTS[PYTHON_PYLINT]}" =~ "json" ]] && LINTER_OPTS[PYTHON_PYLINT]+=" --output-format=json"
+  [[ ! "${LINTER_OPTS[DOCKERFILE_HADOLINT]}" =~ "json" ]] && LINTER_OPTS[DOCKERFILE_HADOLINT]+=" --format json"
+}
```

Comment:
Bug maybe when I run on toolsweet:
https://github.com/scriptsrc/toolsweet/runs/2258624568?check_suite_focus=true

```
Usage: pylint [options]

pylint: error: no such option: --format
```

Not exactly sure why `--format` was passed to pylint.

My `add-bandit` branch is also failing as lintly is complaining it's not being passed proper json.

Reply:
nevermind... this seems like a bug I introduced in my add-bandit branch because I copied and pasted line 34 but only replaced PYTHON_PYLINT in one place instead of both places.